src/nimlsp.nim

```diff
@@ -232,7 +233,7 @@ while true:
               openFiles[fileuri].fingerTable[rawLine].utf16to8(rawChar)
             )
             debugEcho "Found suggestions: ",
-              suggestions[0..(if suggestions.len > 10: 10 else: suggestions.high)],
+              (if suggestions.len > 0 : suggestions[0..(if suggestions.len > 10: 10 else: suggestions.high)]  else: newSeq[Suggest]() ),
```

Comment:
This shouldn't be required a slice `[0..0]` of an empty seq is still just an empty seq.

Reply:
Error: unhandled exception: index out of bounds, the container is empty [IndexError]
https://play.nim-lang.org/#ix=2nhy

Reply:
Oh sorry, I mean that `[0..suggestions.high]` returns an empty sequence: https://play.nim-lang.org/#ix=2nhA

Reply:
then I dont know now , I always see `IndexError` during debug